now.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": 2,
+    "name": "SmartComm",
+    "builds": [
+      {
+        "src": "package.json",
+        "use": "@now/static-build",
+        "config": { "distDir": "build" }
+      }
+    ],
+    "routes": [
+      {
+        "src": "/assets/(.*)",
+        "headers": { "cache-control": "s-maxage=31536000,immutable" },
+        "dest": "/static/$1"
+      },
+      { "src": "/favicon.ico", "dest": "/favicon.ico" },
+      { "src": "/asset-manifest.json", "dest": "/asset-manifest.json" },
+      { "src": "/manifest.json", "dest": "/manifest.json" },
+      { "src": "/precache-manifest.(.*)", "dest": "/precache-manifest.$1" },
+      {
+        "src": "/service-worker.js",
+        "headers": { "cache-control": "s-maxage=0" },
+        "dest": "/service-worker.js"
+      },
+      { "src": "/sockjs-node/(.*)", "dest": "/sockjs-node/$1" },
+      {
```

Comment:
! this is breaking the website
it redirects every js file to the html

1. ideally, this should not be needed (especially if you are using `nextjs` & `now`
2. at least the js & other static files need to be directed to their correct url (the code needs to be added before doing this match all route):
`{"src": "/static/(.*)", "dest": "/static/$1"}`